ichnaea/db.py

```diff
@@ -19,29 +21,33 @@
 from sqlalchemy.sql import func, select
 from sqlalchemy.sql.expression import Insert
 
-from ichnaea.config import (
-    ALEMBIC_CFG,
-    DB_LIBRARY,
-    DB_DDL_URI,
-    DB_RW_URI,
-    DB_RO_URI,
-)
+from ichnaea.conf import settings
+
 
 DB_TYPE = {
-    None: DB_DDL_URI,
-    'ddl': DB_DDL_URI,
-    'ro': DB_RO_URI,
-    'rw': DB_RW_URI,
+    'ddl': settings('db_ddl_uri'),
+    'ro': settings('db_readonly_uri'),
+    'rw': settings('db_readwrite_uri'),
 }
 
 DB_TRANSPORTS = {
-    'default': DB_LIBRARY,
+    'default': settings('db_library'),
```

Comment:
Note that this gets set at module-load time and won't update if we do a `config_override` in the tests.